pkg/security/test/jwt_test.go

```diff
@@ -83,3 +86,45 @@ func TestChain(t *testing.T) {
 
 	Expect(security.VerifySignature(signature3, sc3.GetCABundle(), SpiffeID3)).To(BeNil())
 }
+
+func TestJWTPerformance(t *testing.T) {
```

Comment:
Probably for this kind tests we can use `testing.B` and run these tests in a separate job on CI. @edwarnicke , @lobkovilya , @haiodo What do you think?

Reply:
The name of test might be confusing, but in real it works about 200ms :)

Reply:
> The name of test might be confusing, but in real it works about 200ms :)

Yes, I've checked before putting a comment :) 
The main idea is to write a more pure test to track performance vie testing.B.